### ISSUE TITLE
Folder copy not working if folder already exist in destination.

### ISSUE THREAD

**Issue by afry23** [fman-community/fman#735]
Hi,

say you have a folder _C:\Temp\Foo_ with files in it (like test1.txt and test2.txt) and another folder _C:\Temp\Bar\Foo_ (which is empty).

When you try to copy the folder _C:\Temp\Foo_ to _C:\Temp\Bar_ (overwriting the existing _C:\Temp\Bar\Foo_) the existing folder won't be overwritten. Instead the files from folder C:\Temp\Foo will be copied to  _C:\Temp\Bar_. The content of _C:\Temp\Bar_ will look like:
```
C:\Temp\Bar\Foo
C:\Temp\Bar\test1.txt
C:\Temp\Bar\test2.txt
```

What I expect is:
```
C:\Temp\Bar\Foo\test1.txt
C:\Temp\Bar\Foo\test2.txt
```

OS: Windows 10
fman version: 1.7.1

Would be nice if you could fix this please.

Best regards,
Andy

**Comment by mherrmann** [comment]
Sorry, I cannot reproduce this. I get `Bar\Foo\test1.txt` and `Bar\Foo\test2.txt` as expected. I tried on both Windows and Linux.

**Comment by afry23** [comment]
I tried again and noticed the following:
With CTRL-C (in left pane) and CTRL-V (in right pane) i get the described behavior.
When i use CTRL-SHIFT-P and select _Copy_ it works as expected.

**Comment by mherrmann** [comment]
Fixed in [fman 1.7.2](https://fman.io/changelog#v1.7.2).
